Microsoft.Toolkit.Uwp.Connectivity/Microsoft.Toolkit.Uwp.Connectivity.csproj

```diff
@@ -1,8 +1,11 @@
 <Project Sdk="MSBuild.Sdk.Extras">
 
   <PropertyGroup>
-    <TargetFrameworks>uap10.0.17763</TargetFrameworks>
-    <Title>Windows Community Toolkit Devices</Title>
+    <TargetFramework>uap10.0.17763</TargetFramework>
```

Comment:
This would un-do the effect of #3651. We are using TargetFramework**s** for Uno compatibility.
@jeromelaban is this still the case with Uno, or are we ok with this change?

Reply:
I see. But `TargetFrameworks` change is not needed in the repo for what he's trying to achieve.

Then again, I understand that this is more of a convenience. It also makes the first forked commit have no changes in project files if only `Directory.Build.targets` is changed. This results in a simpler patch tree.

I've looked at the fork, seems it has not been updated in a while.

@michael-hawker

Is there any plans to support WinUI v2 once v3 releases? If so, we could multi-target for a while instead of maintaining multiple branches. Then, this change is just unnecessary churn, I can revert it.

Reply:
I see. But the above change is not needed in the repo for what he's trying to achieve.

Then again, I understand that this is more of a convenience. It also makes the first forked commit have no changes in project files if only `Directory.Build.targets` is changed. This results in a simpler patch tree.

I've looked at the fork, seems it has not been updated in a while.

@michael-hawker

Is there any plans to support WinUI v2 once v3 releases? If so, we could multi-target for a while instead of maintaining multiple branches. Then, this change is just unnecessary churn, I can revert it.

Reply:
@azchohfi Indeed, this is easier to manage for our Uno fork that adds additional target frameworks. It avoids merge conflicts when we're tracking main.

Reply:
I'll revert these changes!